src/common/datetime.ts

```diff
@@ -38,3 +40,15 @@ export const isoToDateOnlyString = (iso = ""): string =>
     day: "numeric",
     year: "numeric",
   });
+
+/**
+ * Returns a nicely formatted date-time string with GMT+08:00 timezone.
+ * @param iso "2020-09-28T06:15:00Z"
+ * @returns "9/28/20 6:15:00 AM GMT+08:00"
+ */
+export const parseDateTime = (dateString: string | undefined): string =>
+  dateString
+    ? `${moment
+        .tz(dateString, "Asia/Singapore")
+        .format("M/D/YY h:mm:ss A")} GMT+08:00`
+    : "";
```

Comment:
Should we use an implementation similar to this? Heard that the `moment` library is considered legacy already.

https://github.com/Notarise-gov-sg/healthcert-renderer/blob/d8a13aa7c69fd9a8788eb39a30c88c49606e58f6/src/util/datetime.ts#L20-L53

Reply:
Also, I realised we didn't check whether the provided string is a valid ISO datetime string. Should we add that check and return the correct error?

Reply:
Anyway, if we still have a lot of usage of the `moment` library, I guess can just leave it as it is

Reply:
yes, it's also use for child dob age calculation.

Reply:
Hmm actually this function is a duplicate of the one above?

https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/171b8984888944e7ed7dc9accf547ba2ebc0c451/src/common/datetime.ts#L7-L20

Reply:
nope.. this function is use in email date display format.